en/includes/guides/authentication/sso-integrations/add-slack-template.md

```diff
@@ -0,0 +1,72 @@
+# Log in to Slack using the {{ product_name }}
+
+This page guides you through using {{ product_name }} to log in to Slack.
+
+!!! note
+    Ensure you have a Slack Workspace account and your account supports SSO by referring to the [Slack documentation](https://slack.com/intl/en-gb/help/articles/203772216-SAML-single-sign-on){:target="_blank"}.
+
+## Create the Slack Service Provider
+
+Follow the steps given below to register the Slack application in {{ product_name }}.
+
+1. On the {{ product_name }} Console, go to **Applications**.
+
+2. Click **New Application** and select **Slack** from **SSO Integrations** section.
+
+    ![Select app type in the {{ product_name }}]({{base_path}}/assets/img/guides/authentication/sso-integrations/common/add-app.png){: width="600" style="border: 0.3px solid lightgrey;"}
+
+3. Provide the Slack issuer name, domain name and a name for the application.
+
+    !!! info "Issuer Name"
+        By default, Slack uses `https://slack.com` as the Issuer Name. You only need to modify this value if creating multiple Slack SSO applications. If you choose `Slack Workspace URL` as the Issuer Name, ensure that Slack's configuration is updated accordingly.
+
+    !!! info "Domain Name"
+        Slack domain name is the first part of your Slack workspace URL. If your Slack workspace URL is `https://myorg.slack.com`, enter `myorg` as the Domain Name. Follow Slack's guidelines to [locate your Slack URL](https://slack.com/intl/en-gb/help/articles/221769328-Locate-your-Slack-URL-or-ID){:target="_blank"}
+
+    ![Select slack workspace app in the {{ product_name }}]({{base_path}}/assets/img/guides/authentication/sso-integrations/slack/add-slack-app.png){: width="600" style="border: 0.3px solid lightgrey;"}
+
+4. Click **Create** to complete the registration.
+
+-----
+
+## Configure Slack
+
+1. Follow [Slack's documentation](https://slack.com/intl/en-gb/help/articles/212681477-Sign-in-to-Slack){:target="_blank"} to log in to your workspace.
```

Comment:
```suggestion
1. Follow [Slack documentation](https://slack.com/intl/en-gb/help/articles/212681477-Sign-in-to-Slack){:target="_blank"} to log in to your workspace.
```

Reply:
https://github.com/wso2/docs-is/pull/4893/commits/4331f958e1550c4ea18946cbbc29e70d46d2e07b